lisa/tests/scheduler/numa_behaviour.py

```diff
@@ -0,0 +1,132 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2019, Linaro and contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import os.path
+
+import pandas as pd
+
+from lisa.wlgen.rta import Periodic
+from lisa.tests.base import ResultBundle, RTATestBundle
+from lisa.utils import ArtifactPath
+from lisa.trace import requires_events
+from lisa.target import Target
+from lisa.trace import FtraceCollector
+from lisa.datautils import df_filter_task_ids
+
+class NUMABehaviour(RTATestBundle):
+    """
+    Abstract class for NUMA related scheduler testing.
+
+    This class provides :meth:`test_task_placement` to validate the basic
+    behaviour of NUMA related scheduling decisions. 
+    """
+
+    @classmethod
+    def _from_target(cls, target: Target, *, res_dir: ArtifactPath = None, ftrace_coll: FtraceCollector = None) -> 'NUMABehaviour':
```

Comment:
AFAICS this definition of `_from_target` is redundant with what is inherited from `RTATestBundle` so removing should still work (let me know if it does not work so I can fix `RTATestBundle`).